lib/internal/http.js

```diff
@@ -26,8 +31,32 @@ function resetCache() {
   utcCache = undefined;
 }
 
+let traceEventId = 0;
+
+function getNextTraceEventId() {
+  return ++traceEventId;
+}
+
+function isTraceHTTPCategory() {
```

Comment:
I believe `isTraceHttpEnabled` is a better naming convention and syncs better with `isTraceCategoryEnabled`

Reply:
Thanks ! Done !